sys/dev/athn/if_athn_usb.c

```diff
@@ -1588,12 +1717,17 @@ athn_usb_read_rom(struct athn_softc *sc)
 	eep = sc->eep;
 	addr = AR_EEPROM_OFFSET(sc->eep_base);
 	for (i = 0; i < sc->eep_size / 16; i++) {
+		DELAY(3000);
 		for (j = 0; j < 8; j++, addr += 4)
 			addrs[j] = htobe32(addr);
 		error = athn_usb_wmi_xcmd(usc, AR_WMI_CMD_REG_READ,
 		    addrs, sizeof(addrs), vals);
+		DELAY(3000);
 		if (error != 0)
+		{
```

Comment:
Code formatting. '{' shall be in the line above.

Reply:
I would leave it for now. We should use formatting tool at the end of implementing driver for all changes that we made.